Makefile

```diff
@@ -30,15 +30,19 @@ endif
 
 PROPER_REBAR := .$(SEP)rebar
 
+REBAR3_URL := https://s3.amazonaws.com/rebar3/rebar3
+REBAR3 ?= $(shell which rebar3 || which .$(SEP)rebar3 || \
+            (wget $(REBAR3_URL) && chmod +x rebar3 && echo .$(SEP)rebar3))
+
 default: compile
 
 all: compile dialyzer doc test
 
 compile:
-	$(PROPER_REBAR) compile
+	$(REBAR3) compile
 
 dialyzer: .plt/proper_plt compile
-	dialyzer -n -nn --plt $< -Wunmatched_returns ebin
+	dialyzer -n -nn --plt $< -Wunmatched_returns _build/default/lib/proper/ebin
```

Comment:
rebar3 has a dialyzer command that should take care of building the plt and including the correct beams

Reply:
I know about the `rebar3 dialyzer` command but I do not like it.  It requires having an installed Erlang/OTP system (which I do not have) in order to find `erts` as an application to include in the PLT.  I prefer calling `dialyzer` directly, and I do not see the point in not using `make` for these things.  Note that calling Dialyzer and running the tests is for PropEr developers only, not for regular users.